lib/src/cpp/utils/ErrorHandlerXMLManager.hpp

```diff
@@ -16,8 +16,8 @@
  * @file
  */
 
-#ifndef _FAST_DDS_QOS_PROFILES_MANAGER_UTILS_PARSE_XML_ERROR_HANDLER_HPP_
-#define _FAST_DDS_QOS_PROFILES_MANAGER_UTILS_PARSE_XML_ERROR_HANDLER_HPP_
+#ifndef _FAST_DDS_QOS_PROFILES_MANAGER_UTILS_XML_MANAGER_ERROR_HANDLER_HPP_
```

Comment:
You have altered the order. It should be:

```suggestion
#ifndef _FAST_DDS_QOS_PROFILES_MANAGER_UTILS_ERROR_HANDLER_XML_MANAGER_HPP_
```